test/dfusion_multi_safes.js

```diff
@@ -731,7 +710,12 @@ contract("GnosisSafe", function (accounts) {
       }
 
       const requestWithdrawalTransaction = await buildRequestWithdraw(masterSafe.address, withdrawals)
-      await execTransaction(masterSafe, lw, requestWithdrawalTransaction, "request withdrawal for all brackets")
+      await execTransaction(
+        masterSafe,
+        safeOwner.privateKey,
+        requestWithdrawalTransaction,
+        "request withdrawal for all brackets"
```

Comment:
I really wonder where this string came from. Maybe you can remove it, since you removed a similar one somewhere else in the code?